boards/PDM/Src/App/states/App_InitState.c

```diff
@@ -21,7 +24,7 @@ const struct State *App_GetInitState()
 {
     static struct State init_state = {
         .name         = "INIT",
-        .run_on_enter = InitStateRunOnEntry,
+        .run_on_entry = InitStateRunOnEntry,
```

Comment:
Change all instances of `run_on_enter` to `run_on_entry` to better fit the function names